dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -277,6 +277,11 @@ export const Card = ({
 		return <Snap snapData={snapData} />;
 	}
 
+	// Decide what type of image to show, main media, avatar or none
+	let imageType: 'mainmedia' | 'avatar' | undefined;
+	if (imageUrl && avatarUrl) imageType = 'avatar';
```

Comment:
Do we only want to show an avatar if there is an image, is that right?

If not, you can use my helper function above as-is.